config/openshift-olm/rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
wouldn't that install the privileged mode unconditionally? I'd say it should be installed only if `privileged` is true in FlowCollector, no?

Reply:
I don't think so @jotak this just creating service account the error we get when enable privileged mode was
```sh
Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
```
I don't know how if that config knob ever worked before seems like day1 bug ?

Reply:
I just did the test to switch on privileged mode (not with your PR, but from `main` ) : it works as expected, ie. the agent pods don't have the CAP_* stuff anymore but just have the `privileged` flag. So that doesn't look like a day1 bug, maybe that's something that is additionally needed with your changes?

Anyway, my point wasn't about that, but it's that this role binding should only be installed when `privileged` is `true`, and removed when set back to `false`, so something to reconcile probably somewhere here: https://github.com/netobserv/network-observability-operator/blob/main/controllers/ebpf/internal/permissions/permissions.go